tests: mimxrt1050_evk : samples/sensor/qdec : build failure

**Describe the bug**
samples/sensor/qdec build failure



Please also mention any information which could help others to understand
the problem you're facing:
 - What target platform are you using?
 mimxrt1050_evk
 - What have you tried to diagnose or workaround this issue?
 git bisect
 - Is this a regression? If yes, have you been able to "git bisect" it to a
   specific commit?
 
```
e6d8ed2e0d99bb25e9c4002b2a14c8c3a15e9e9b is the first bad commit
commit e6d8ed2e0d99bb25e9c4002b2a14c8c3a15e9e9b
Author: Gerson Fernando Budke <nandojve@gmail.com>
Date:   Wed Jun 19 21:49:29 2024 +0200

    samples: sensor: qdec: Fix wrong harness
    
    The #73619 which fixes the atmel,sam-tc-qdec sensor implementation
    introduced agregression on the sample test removing the harness
    console. That changed the test to use the default ztest behaviour
    which is incompatible with the test outpout. This restore the
    harness to the default console value.
    
    Fixes: #74531
    
    Signed-off-by: Gerson Fernando Budke <nandojve@gmail.com>

 samples/sensor/qdec/sample.yaml | 13 +++++++++++++
 1 file changed, 13 insertions(+)
```

**To Reproduce**
Steps to reproduce the behavior:
```
west twister -p mimxrt1050_evk -s samples/sensor/qdec/sample.sensor.qdec_sensor
```

**Expected behavior**
build pass

**Impact**
sensor supporting

**Logs and console output**

```
-- Board: mimxrt1050_evk, Revision: hyperflash, qualifiers: mimxrt1052
-- Found host-tools: zephyr 0.16.8 (/opt/toolchains/zephyr-sdk-0.16.8)
-- Found toolchain: zephyr 0.16.8 (/opt/toolchains/zephyr-sdk-0.16.8)
-- Found Dtc: /opt/toolchains/zephyr-sdk-0.16.8/sysroots/x86_64-pokysdk-linux/usr/bin/dtc (found suitable version "1.6.0", minimum required is "1.4.6") 
-- Found BOARD.dts: /home/jenkins/agent/workspace/bitbucket_build_platform/zephyr/boards/nxp/mimxrt1050_evk/mimxrt1050_evk.dts
-- Found devicetree overlay: /home/jenkins/agent/workspace/bitbucket_build_platform/zephyr/boards/nxp/mimxrt1050_evk/mimxrt1050_evk_mimxrt1052_hyperflash.overlay
-- Found devicetree overlay: /home/jenkins/agent/workspace/bitbucket_build_platform/zephyr/samples/sensor/qdec/boards/mimxrt1050_evk.overlay
devicetree error: /home/jenkins/agent/workspace/bitbucket_build_platform/zephyr/samples/sensor/qdec/boards/mimxrt1050_evk.overlay:41 (column 1): parse error: undefined node label 'touch_controller'
-- In: /home/jenkins/agent/workspace/bitbucket_build_platform/zephyr/twister-out/mimxrt1050_evk/samples/sensor/qdec/sample.sensor.qdec_sensor/zephyr, command: /usr/bin/python;/home/jenkins/agent/workspace/bitbucket_build_platform/zephyr/scripts/dts/gen_defines.py;--dts;/home/jenkins/agent/workspace/bitbucket_build_platform/zephyr/twister-out/mimxrt1050_evk/samples/sensor/qdec/sample.sensor.qdec_sensor/zephyr/zephyr.dts.pre;--dtc-flags;'';--bindings-dirs;/home/jenkins/agent/workspace/bitbucket_build_platform/zephyr/dts/bindings;--header-out;/home/jenkins/agent/workspace/bitbucket_build_platform/zephyr/twister-out/mimxrt1050_evk/samples/sensor/qdec/sample.sensor.qdec_sensor/zephyr/include/generated/zephyr/devicetree_generated.h.new;--dts-out;/home/jenkins/agent/workspace/bitbucket_build_platform/zephyr/twister-out/mimxrt1050_evk/samples/sensor/qdec/sample.sensor.qdec_sensor/zephyr/zephyr.dts.new;--edt-pickle-out;/home/jenkins/agent/workspace/bitbucket_build_platform/zephyr/twister-out/mimxrt1050_evk/samples/sensor/qdec/sample.sensor.qdec_sensor/zephyr/edt.pickle;--edtlib-Werror;--vendor-prefixes;/home/jenkins/agent/workspace/bitbucket_build_platform/zephyr/dts/bindings/vendor-prefixes.txt
CMake Error at cmake/modules/dts.cmake:296 (message):
  gen_defines.py failed with return code: 1
Call Stack (most recent call first):
  cmake/modules/zephyr_default.cmake:132 (include)
  share/zephyr-package/cmake/ZephyrConfig.cmake:75 (include)
  share/zephyr-package/cmake/ZephyrConfig.cmake:97 (include_boilerplate)
  cmake/package_helper.cmake:126 (find_package)
```

**Environment (please complete the following information):**
 - OS: (e.g. Linux)
 - Toolchain (e.g Zephyr SDK, ...)
 - Commit SHA or Version used: v3.7.0-rc1-243-gaf57038bb053

--- a/samples/sensor/qdec/boards/mimxrt1050_evk.overlay
+++ b/samples/sensor/qdec/boards/mimxrt1050_evk.overlay
@@ -37,11 +37,6 @@
 	status = "disabled";
 };
 
-/* disable to avoid encoder emulater conflict */
-&touch_controller {
-	status = "disabled";
-};
-
 &qdec1 {
 	status = "okay";
 	pinctrl-0 = <&pinmux_qdec1>;

